src/cdef_tmpl_8.rs

```diff
@@ -922,22 +918,26 @@ unsafe extern "C" fn cdef_dsp_init_x86(c: *mut Dav1dCdefDSPContext) {
     (*c).fb[1] = Some(dav1d_cdef_filter_4x8_8bpc_sse4);
     (*c).fb[2] = Some(dav1d_cdef_filter_4x4_8bpc_sse4);
 
-    if flags & DAV1D_X86_CPU_FLAG_AVX2 == 0 {
-        return;
-    }
+    cfg_if!{
+        if #[cfg(target_arch = "x86_64")] {
+            if flags & DAV1D_X86_CPU_FLAG_AVX2 == 0 {
```

Comment:
Since there's no `else` in this `cfg_if!` block, it might be cleaner to use a regular `#[cfg]` attribute here and the similar cases you fix in this PR:

```rust
#[cfg(target_arch = "x86_64")]
{
    // ...
}
```